src/screens/ProfileScreen/DetailScreen/ExtensionRequest.tsx

```diff
@@ -94,7 +91,9 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: 'white',
-    border: '2px solid black',
```

Comment:
Why did we change into separate property?

Reply:
You can google about this.